### PR TITLE
2 - Create Components

### LINKED ISSUE #2: Create components
Make the app to be in different components instead of only in App.js

### PR DESCRIPTION
**Pull request by COANelmejor**
Converted to individual components, so the code is more modular and maintainable.

Solves #2

### CODE REVIEW

**Review by UdanPrg — APPROVED** [review]
Visto bueno!

File: src/components/cards/WorkTimerSetup.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import Button from "../../components/bites/Button";
+import CronometerTimeView from "../../utils/CronometerTimeView";
+
+function WorkTimerSetup({ setWorkTimeButton, workTime }) {
+  return (
+    <div className="WorkTime">
+      <h3 className="text-3xl font-bold p-2">Work Time</h3>
+      <div className="flex justify-center items-center">
+        <div className="flex flex-col col-25">
+          <Button onClickFn={setWorkTimeButton} onClickParams={-1} text="-0.1s" color="red" />
```
Comment (UdanPrg) [inline src/components/cards/WorkTimerSetup.js]: This multiples buttons declarations, could be an array, coverted 'onClickParams' and 'text' to object params

File: src/components/cards/WorkTimerSetup.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          <Button onClickFn={setWorkTimeButton} onClickParams={-10} text="-1s" color="red" />
```
Comment (UdanPrg) [inline src/components/cards/WorkTimerSetup.js]: This multiples buttons declarations, could be an array, coverted 'onClickParams' and 'text' to object params

File: src/components/cards/WorkTimerSetup.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          <Button onClickFn={setWorkTimeButton} onClickParams={-6000} text="-10m" color="red" />
+          <Button onClickFn={setWorkTimeButton} onClickParams={-36000} text="-1h" color="red" />        
+        </div>
+        <div className="flex flex-col col-50">
+          <h3 className="text-5xl font-bold p-2">
+            {CronometerTimeView(workTime)}
+          </h3>
+        </div>
+        <div className="flex flex-col col-25">
+          <Button onClickFn={setWorkTimeButton} onClickParams={1} text="+0.1s" color="green" />
```
Comment (UdanPrg) [inline src/components/cards/WorkTimerSetup.js]: This multiples buttons declarations, could be an array, coverted 'onClickParams' and 'text' to object params

File: src/components/cards/RestTimerSetup.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+import CronometerTimeView from '../../utils/CronometerTimeView';
+
+function RestTimerSetup({ restTime, setRestTimeButton }) {
+  return (
+    <div className="RestTime">
+      <h3 className="text-3xl font-bold p-2">Rest Time</h3>
+      <div className="flex justify-center items-center">
+        <div className="flex flex-col col-25">
+          <Button onClickFn={setRestTimeButton} onClickParams={-1} text="-0.1s" color="red" />
```
Comment (UdanPrg) [inline src/components/cards/RestTimerSetup.js]: This multiples buttons declarations, could be an array, coverted 'onClickParams' and 'text' to object params

File: src/components/cards/RestTimerSetup.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+          <Button onClickFn={setRestTimeButton} onClickParams={-6000} text="-10m" color="red" />
+          <Button onClickFn={setRestTimeButton} onClickParams={-36000} text="-1h" color="red" />
+        </div>
+        <div className="flex flex-col col-50">
+          <h3 className="text-5xl font-bold p-2">
+            {CronometerTimeView(restTime)}
+          </h3>
+        </div>
+        <div className="flex flex-col col-25">
+          <Button onClickFn={setRestTimeButton} onClickParams={1} text="+0.1s" color="green" />
```
Comment (UdanPrg) [inline src/components/cards/RestTimerSetup.js]: This multiples buttons declarations, could be an array, coverted 'onClickParams' and 'text' to object params

**Review by UdanPrg — CHANGES_REQUESTED** [review]

File: src/components/cards/RestTimerSetup.js (hunk 5 above)
Comment (COANelmejor) [reply src/components/cards/RestTimerSetup.js]: Updated in 834ebe8

File: src/components/cards/RestTimerSetup.js (hunk 4 above)
Comment (COANelmejor) [reply src/components/cards/RestTimerSetup.js]: Updated in 834ebe8

File: src/components/cards/WorkTimerSetup.js (hunk 3 above)
Comment (COANelmejor) [reply src/components/cards/WorkTimerSetup.js]: Updated in 834ebe8

File: src/components/cards/WorkTimerSetup.js (hunk 2 above)
Comment (COANelmejor) [reply src/components/cards/WorkTimerSetup.js]: Updated in 834ebe8

File: src/components/cards/WorkTimerSetup.js (hunk 1 above)
Comment (COANelmejor) [reply src/components/cards/WorkTimerSetup.js]: Updated in 834ebe8

**Review by UdanPrg — APPROVED** [review]
Success PR

**Pull request merged**